actor_bundles/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
Could we just add `actor_bundles` to the root `.gitignore`? In general, it's a bit surprising to suddenly have such directory. Maybe we could use `target/`?

Reply:
Moved to `target/actor_bundles/`